app.js

```diff
@@ -18,19 +18,28 @@ let mySQLSessionConfig = {
   port: config.port,
   user: config.user,
   password: config.password,
-  database: config.database
+  database: config.database,
+  nodeEnv: config.nodeEnv
```

Comment:
You wouldn't need `config.nodeEnv` here, as that isn't a valid option for creating a MySQLStore. You can safely remove this, as all we need to use`config.nodeEnv` for right now is the production check to use the MySQLStore